runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1920,14 +1917,15 @@ Java_java_lang_Class_getNestMembersImpl(JNIEnv *env, jobject recv)
 			PUSH_OBJECT_IN_SPECIAL_FRAME(currentThread, resultObject);
 			J9Class *nestMember = vmFuncs->internalFindClassUTF8(currentThread, J9UTF8_DATA(nestMemberName), J9UTF8_LENGTH(nestMemberName), classLoader, J9_FINDCLASS_FLAG_THROW_ON_FAIL);
 			resultObject = POP_OBJECT_IN_SPECIAL_FRAME(currentThread);
-
+			nestMember = VM_VMHelpers::currentClass(nestMember);
+			J9Class* nestHost = nestMember->nestHost;
 			if (NULL == nestMember) {
```

Comment:
It's not safe to dereference `nestMember` if it might be `NULL`.
Also, space should be before, not after `*`.

Reply:
Fixed.